src/thread_manager.h

```diff
@@ -11,7 +11,7 @@ namespace internal {
 
 class ThreadManager {
  public:
-  ThreadManager(int num_threads)
+  explicit ThreadManager(int num_threads)
```

Comment:
I disagree with this change. A `ThreadManager` shouldn't be implicitly construct from an int.

Reply:
But that's exactly what the explicit keyword states!

Reply:
Oh, I read the diff backwards somehow.

thank you for this fix.